manifests/init.pp

```diff
@@ -51,7 +51,7 @@
   }
 
   if $include_api_collection {
-    if ($pe_username == undef) or ($pe_password == undef) or ($pe_console == undef) {
+    if ($pe_username == undef) or ($pe_console == undef) {
```

Comment:
Is the username required if they are providing a token? Should it be something like if (($pe_username == undef) and ($pe_token == undef)) or ($pe_console == undef)?

Reply:
Good point. Rejiggered it to get rid of the `pe_console` check that we don't need to do anymore either.